server.js

```diff
@@ -19,13 +19,16 @@ server.use(express.static(__dirname));
 server.use(bodyparser.json());
 
 server.set('view engine', 'ejs');
+server.set('views', __dirname + '/app/views');
 
 var api = require('./api');
+var app = require('./app');
 
 server.get('/', function (req, res) {
     res.sendFile('./index.html');
 });
 
+server.use('/app', app);
```

Comment:
Why do you want an `/app`? Or if you still want that, maybe redirect `/` to `/app`?